statsd/fnv1a.go

```diff
@@ -0,0 +1,39 @@
+package statsd
+
+const (
+	// FNV-1a
```

Comment:
Curious: is there any reason for not using https://golang.org/src/hash/fnv/fnv.go?s=1294:1319#L40?

Reply:
Unfortunately, the way the standard hash API is exposed automatically leads to dynamic allocations. This is not desirable in our case since it's pretty common to have calls to these library methods in application hot paths.